src/test/groovy/pl/edu/agh/iet/katabank/AccountTest.groovy

```diff
@@ -16,4 +16,25 @@ class AccountTest extends Specification{
         where:
         anotherAccount = new Account(customer)
     }
+
+    def "try to withdraw a negative amount"() {
+        when:
+        account.withdraw(new BigDecimal(-1))
```

Comment:
In groovy java.math.BigDecimal is the default type for non-integer floating types. You don't have to create them using the new keyword.

Reply:
Corrected in all test cases.